apps/docmost/config.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "Docmost",
+    "id": "docmost",
+    "available": true,
+    "short_desc": "An open-source collaborative wiki and documentation software",
+    "author": "docmost",
+    "port": 9713,
+    "categories": ["data", "utilities"],
+    "description": "Docmost is an open-source alternative to the likes of Notion and Confluence. Whether you're managing a wiki, a knowledge base, or extensive project documentation, Docmost provides the tools you need to create, collaborate, and share knowledge effortlessly.",
+    "tipi_version": 5,
+    "version": "0.2.5",
+    "source": "https://github.com/docmost/docmost",
+    "website": "https://docmost.com/",
+    "exposable": true,
+    "force_expose": false,
+    "form_fields": [
+      {
+        "type": "password",
+        "label": "App Secret",
+        "max": 64,
+        "min": 12,
+        "required": true,
+        "env_variable": "DOCMOST_APP_SECRET"
+      },
```

Comment:
I am a bit confused here, according to docmost docker compose seems like that doesn't need to be entered by the user, a random field seems more appropriate.